examples/tracing/cache_miss.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
```

Comment:
I think it'd be ok to name this cachemiss.py. If you look at /tools, and also Linux /usr/bin, underscores are rare.